docs/guides/community-inspirations.md

```diff
@@ -14,7 +14,7 @@ head:
 This page is a tribute to our Community who have been using WebContainers over the past two years - and who have been building with WebContainer API since it was released in June 2022 in a private alpha. Here are some of the projects that are powered by it.
 
 ::: tip Get your project featured!
-Have a project you'd like us to showcase here and on our social media? [Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
+Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```

Comment:
Actually is this link correct? This is not on Discord but GitHub? 🤔

Reply:
```suggestion
Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```